x-pack/winlogbeat/module/security/ingest/security_standard.yml

```
@@ -3716,6 +3716,10 @@ processors:
           ctx.process.put("args", al);
           ctx.process.put("command_line", ctx.winlog.event_data.CommandLine);
         }
+        def args = ctx?.process.args;
+          if (args != null && args != "") {
+            ctx.process.args_count = ctx.process.args.length;
+        }
```

```suggestion
        if (ctx.process?.args instanceOf List) { ctx.process.args_count = ctx.process.args.size(); }
```

This will ensure process.args is a non null List before trying to access the `.size()`.